device-client/singlethreaded/x509-authentication/linux/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 3.10)
+project(iothub_ll_client_x509_sample_linux)
+
+if (NOT "${use_amqp}" AND NOT "${use_mqtt}" AND NOT "${use_http}")
+    message("CMAKE WARNING: No protocol has been set. 
+    Defaulting to link mqtt, amqp, and http.
+    To only link a specific protocol, add the cmake option '-D use_<protocol>=ON'")
+    set(use_mqtt ON)
+    set(use_amqp ON)
+    set(use_http ON)
+endif()
+
+set(AZUREIOT_INC_FOLDER "/usr/include/azureiot" "/usr/include/azureiot/inc")
+
+set(sample_application_c_files
+    ${PROJECT_SOURCE_DIR}/../iothub_ll_client_x509_sample.c
+    ${PROJECT_SOURCE_DIR}/../../../../tools/.certs/certs.c
+)
+
+set(sample_application_h_files
+)
+
+add_executable(${PROJECT_NAME} 
+    ${sample_application_c_files} 
+    ${sample_application_h_files}
+)
+
+add_definitions(-DSET_TRUSTED_CERT_IN_SAMPLES)
```

Comment:
>SET_TRUSTED_CERT_IN_SAMPLES [](start = 18, length = 27)

Why would we use this?  You will make people have to decide if they need the trusted cert or not.

Reply:
fair. For linux I will make it default.